plugins/BEdita/Core/config/Migrations/20170301144829_RemovePluralized.php

```diff
@@ -0,0 +1,73 @@
+<?php
+use Migrations\AbstractMigration;
+
+class RemovePluralized extends AbstractMigration
+{
+
+    public $autoId = false;
+
+    public function up()
+    {
+        $this->table('object_types')
+            ->addColumn('singular', 'string', [
+                'after' => 'name',
+                'comment' => 'singular object type name',
+                'default' => null,
+                'length' => 50,
+                'null' => true,
+            ])
+            ->update();
+
+        $this->query('UPDATE object_types set singular = name');
+        $this->query('UPDATE object_types set name = pluralized');
+
+        $this->table('object_types')
+            ->changeColumn('singular', 'string', ['null' => false])
+            ->update();
```

Comment:
Why not `$this->table('object_types')->renameColumn('name', 'singular')->update()`, then `$this->table('object_types')->renameColumn('pluralized', 'name')->update()` ?